pyctuator/httptrace/fastapi_http_tracer.py

```diff
@@ -7,18 +7,23 @@
 from starlette.responses import Response
 
 from pyctuator.httptrace import TraceRecord, TraceRequest, TraceResponse
-from pyctuator.httptrace.http_tracer import HttpTracer
+from pyctuator.impl.pyctuator_impl import PyctuatorImpl
 
 
 class FastApiHttpTracer:
 
-    def __init__(self, http_tracer: HttpTracer) -> None:
-        self.http_tracer = http_tracer
+    def __init__(self, pyctuator_impl: PyctuatorImpl) -> None:
+        self.pyctuator_impl = pyctuator_impl
+        self.http_tracer = pyctuator_impl.http_tracer
 
     async def record_httptrace(self, request: Request, call_next: Callable) -> Response:
         request_time = datetime.now()
         response: Response = await call_next(request)
         response_time = datetime.now()
+
+        if request.url.path.startswith(self.pyctuator_impl.pyctuator_endpoint_path_prefix):
+            response.headers["Content-Type"] = self.pyctuator_impl.sba_v2_content_type
```

Comment:
I understand now from the commit message why it didn't work before, but I think that might be because the mechanism was the wrong one.
See for example here how you're supposed to add a response header in FastAPI: https://fastapi.tiangolo.com/advanced/response-headers/#use-a-response-parameter

Reply:
That's a good point...